lib/polyfill/patchedmediakeys_20140218.js

```diff
@@ -0,0 +1,800 @@
+      // don't truly know their values from the prefixed API:
+      'persistentState': 'optional',
+      'distinctiveIdentifier': 'optional',
+      // Pretend the requested init data types are supported, since we don't
+      // really know that either:
+      'initDataTypes': cfg.initDataTypes,
+      'sessionTypes': ['temporary']
+    };
+
+    // v0.1b tests for key system availability with an extra argument on
+    // canPlayType.
+    var ranAnyTests = false;
+    if (cfg.audioCapabilities) {
+      for (var j = 0; j < cfg.audioCapabilities.length; ++j) {
+        var cap = cfg.audioCapabilities[j];
+        if (cap.contentType) {
+          ranAnyTests = true;
+          var contentType = cap.contentType.split(';')[0];
+          if (MSMediaKeys.isTypeSupported(this.keySystem, contentType)) {
+            newCfg.audioCapabilities.push(cap);
+            success = true;
+          }
+        }
+      }
+    }
+    if (cfg.videoCapabilities) {
+      for (var j = 0; j < cfg.videoCapabilities.length; ++j) {
+        var cap = cfg.videoCapabilities[j];
+        if (cap.contentType) {
+          ranAnyTests = true;
+          var contentType = cap.contentType.split(';')[0];
+          if (MSMediaKeys.isTypeSupported(this.keySystem, contentType)) {
+            newCfg.videoCapabilities.push(cap);
+            success = true;
+          }
+        }
+      }
+    }
+
+    if (!ranAnyTests) {
+      // If no specific types were requested, we check all common types to find
+      // out if the key system is present at all.
+      success = MSMediaKeys.isTypeSupported(this.keySystem, 'video/mp4');
+    }
+    if (cfg.persistentState == 'required') {
+      if (allowPersistentState) {
+        newCfg.persistentState = 'required';
+      } else {
+        success = false;
+      }
+    }
+
+    if (success) {
+      this.configuration_ = newCfg;
+      return;
+    }
+  }  // for each cfg in supportedConfigurations
+
+  // As per the spec, this should be a DOMException, but
+  // there is not a public constructor for this
+  var unsupportedKeySystemError = new Error('Unsupported keySystem');
+  unsupportedKeySystemError.name = 'NotSupportedError';
+  unsupportedKeySystemError.code = DOMException.NOT_SUPPORTED_ERR;
+  throw unsupportedKeySystemError;
+};
+
+
+/** @override */
+shaka.polyfill.PatchedMediaKeys.v20140218.MediaKeySystemAccess.prototype.
+    createMediaKeys = function() {
+  shaka.log.debug('v20140218.MediaKeySystemAccess.createMediaKeys');
+
+  // Alias
+  var v20140218 = shaka.polyfill.PatchedMediaKeys.v20140218;
+
+  var mediaKeys = new v20140218.MediaKeys(this.keySystem);
+  return Promise.resolve(/** @type {!MediaKeys} */ (mediaKeys));
+};
+
+
+/** @override */
+shaka.polyfill.PatchedMediaKeys.v20140218.MediaKeySystemAccess.prototype.
+    getConfiguration = function() {
+  shaka.log.debug('v20140218.MediaKeySystemAccess.getConfiguration');
+  return this.configuration_;
+};
+
+
+/**
+ * @this {!HTMLMediaElement}
+ * @override
```

Comment:
I don't think override will do what you think here, since this doesn't inherit from anything that defines setMediaKeys, and this isn't a prototype method as defined here.  You may need to add param and return annotations.